### DISCUSSION TITLE
Live Stream for a community member's repo for a chance to get stickers

### DISCUSSION THREAD

**Discussion by BOLT04** [EddieHubCommunity/support#614] [Ideas]
The idea is to have a live stream where a public **GitHub repository from a community member** is chosen, and we all contribute to it during the live stream. Maybe work together on existing issues with the label `help wanted`, or create new issues with ideas. In the end, it would be awesome to have some PRs in and choose some contributors to give away stickers 👍. 

Giving feedback to the repo and to the idea itself would also be cool. so that we Eddie can figure out if this is worth continuing in the future 😃. 

This idea originated in this live stream (timestamp included): https://youtu.be/CNTnHivCI7A?t=4890

Let me know your thoughts on this 😃.

**Comment by vinzvinci** [comment]
That would be awesome, so everyone can get green square :green_square:

**Reply by eddiejaoude** [reply]
Yes, green squares for everyone 👍

**Comment by Vyvy-vi** [comment]
That would be really amazing!
This could also help us give boost to a few projects of our community members(we could perhaps add something like "offer project for live stream contributions" list)

**Reply by eddiejaoude** [reply]
Yes, I am sure there will be lots of people who would love contributions to their projects 👍

**Comment by Nitesh-thapliyal** [comment]
This idea is amazing 🙌All can engage with the project and can learn at the same time.

**Reply by eddiejaoude** [reply]
So true, we can all learn together and get green squares at the same time 🤓

**Comment by eddiejaoude** [comment]
Thank you for raising the discussion @BOLT04 with the information 👍
